README.md

```diff
@@ -1,2 +1,12 @@
-# hw01
-Repository for homework 1
+#Carman Fowler Biography
+##Personal
+###I am from Camden, South Carolina.
+###![Camden Sign](http://maineanencyclopedia.com/wp-content/uploads/camden01.jpg)
```

Comment:
This picture appeared both on the preview and on GitHub. Great job with this code!

Reply:
Totally Agree! was going to ask what the (!) did to the image, but now that I'm looking back at the github cheat sheet I see that that is necessary for the in-line style image. I guess that's why it didn't work for me! 
Seeing the end product, I wonder if there is a way to edit the size and placement of the image with code. It might look nice centered and a little smaller.